lfroms/clouds
# Rebuild Week Day Picker using UIKit & Fix Bugs

### What is the problem being solved in this PR?
This PR rewrites the Week section day picker in UIKit and fixes a bug where it did not correctly scroll to the desired location when tapping an item.

### Related issues or PRs
Resolves #49 

### Why did you choose this approach?
While SwiftUI is great, it does have a few limitations, especially when building very custom controls.

### How to test changes
1. Run the app.
1. Go to the Week section.
1. Tap on various week days.  Ensure that they are always scrolled to.
1. Scroll through the week days. Ensure that the state is updated as expected and that the active day is changed.

### Checklist
